media-video/live555/live555-2016.01.29.recipe

```diff
@@ -23,10 +23,18 @@ REQUIRES="
 	haiku$secondaryArchSuffix
 	"
 
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+"
+
 PROVIDES_devel="
 	live555${secondaryArchSuffix}_devel = $portVersion
 	"
 
+REQUIRES_devel="
+	live555$secondaryArchSuffix == $portVersion base
+"
+
 BUILD_PREREQUIRES="
 	haiku${secondaryArchSuffix}_devel
```

Comment:
This comment goes with the line 39 of the patched recipe. You added  BUILD_REQUIRES, with one identical line in BUILD_PREREQUIRES. I suggest to remove the latter.